### PR TITLE
"Delete Voice Recording" implementation

### PR DESCRIPTION
**Pull request by kcroxford-mb**
mvn test clean 
<redacted> 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.872 s
[INFO] Finished at: 2020-03-12T16:06:50+01:00
[INFO] ------------------------------------------------------------------------

### CODE REVIEW

File: api/src/main/java/com/messagebird/MessageBirdClient.java [hunk 1]
```diff
@@ -1344,7 +1344,29 @@ public RecordingResponse listRecordings(String callID, String legId, final Integ
...
+     * Deletes a voice recording
+     *
+     * @param callID Voice call ID
+     * @param legID Leg ID
+     * @param recordingID
+     * @throws UnauthorizedException if client is unauthorized
+     * @throws GeneralException      general exception
+    */
+    public void deleteRecording(final String callID, final String legID, final String recordingID)
+            throws NotFoundException, GeneralException, UnauthorizedException {
```
Comment (denizkilic) [inline api/src/main/java/com/messagebird/MessageBirdClient.java]: null check and throw exception for call id leg id and recording id as below;
if (callID == null) {
    throw new IllegalArgumentException("Call ID must be specified.");
}

Comment (kcroxford-mb) [reply api/src/main/java/com/messagebird/MessageBirdClient.java]: I've added an example and the requested null checks.

File: examples/src/main/java/ExampleDeleteRecording.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import com.messagebird.MessageBirdService;
+import com.messagebird.MessageBirdServiceImpl;
+import com.messagebird.exceptions.GeneralException;
+import com.messagebird.exceptions.NotFoundException;
+import com.messagebird.exceptions.UnauthorizedException;
+
+public class ExampleDeleteRecording {
+    public static void main(String[] args) {
+
+        if (args.length == 0) {
```
Comment (denizkilic) [inline examples/src/main/java/ExampleDeleteRecording.java]: args.length < 3 would be better

**Comment by denizkilic** [comment]
Hi @kcroxford-mb,
Thanks for your PR. I will close this PR and will merge your changes with small updates because I want to release a new version of SDK today.

**Pull request closed without merging**